test/DebugInfo/line-directive.swift

```diff
@@ -8,7 +8,7 @@ func f() {
   markUsed("Test")
 #sourceLocation(file: "abc.swift", line: 142)
   markUsed("abc again")
-#sourceLocation(file: "/absolute/path/def.swift", line:  142)
+#sourceLocation(file: "//absolute/path/def.swift", line:  142)
```

Comment:
Would it be feasible to leave this as is and add a third path to `C:\absolute-windows` ?

Reply:
If this is still recognized as an absolute path on all platforms, I'm fine with the change, bug we need to document why this path is as weird as it is.

Reply:
No, that would be treated as relative everywhere but Windows, and absolute on Windows.  This path would then be treated as absolute everywhere but Windows, and relative on Windows.